app/models/manageiq/providers/autosde/storage_manager/autosde_client.rb

```diff
@@ -22,10 +22,10 @@ def initialize(username:, password:, host:, port: 443, token: nil, scheme: 'http
     super(configure_openapi_client)
   end
 
-  # generates access methods for OpenApiClient , like StorageSystemApi or CreateVolume
+  # generates access methods for AutosdeOpenapiClient , like StorageSystemApi or CreateVolume
   # usage : <this>.StorageSystemApi.storage_systems_get
-  OpenapiClient.constants.each do |method|
-    clazz = "OpenapiClient::#{method}".constantize
+  AutosdeOpenapiClient.constants.each do |method|
+    clazz = "AutosdeOpenapiClient::#{method}".constantize
```

Comment:
We eventually want to move all these patches to the gem. And also the specs.

Reply:
Agreed, the override for call_api and auto-login looks a little hacky also.  Best to login at the start of a refresh or other operation and move on.  But we can tackle this in a follow-up.